express-api/src/controllers/users/usersSchema.ts

```diff
@@ -0,0 +1,124 @@
+import { z } from 'zod';
```

Comment:
Does having the usersSchema.ts mean that an IUser.ts interface is unnecessary, as I'm assuming this kind of replaces that?